src/txn/txn_ckpt.c

```diff
@@ -941,6 +946,8 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
 	cache->eviction_scrub_limit = 0.0;
 	WT_STAT_CONN_SET(session, txn_checkpoint_scrub_target, 0);
 
+	conn->ckpt_verb_start_time.tv_sec = 0;
```

Comment:
This timer doesn't need to be reset twice, once before "err:" label and once inside "err:" label.
We can reset once at the start of the "err:" label and that should suffice.